docs/maintainers/maintainers-guide.md

```diff
@@ -43,66 +43,72 @@ There are two ways to add repositories to your Repository Insight Page:
 
 The Contributor Insights features allow you to:
 
 - track the commits and PRs of the contributors on your list,
 - filter your contributors by their activity level, including most active, new, and alumni,
-- view the repositories that your contributors have contributed to, the top programming languages they've used, and their time zones.
+- view the repositories your contributors have contributed to, the top programming languages they've used, and their time zones.
 
 ### Why Create a Contributor Insight Page?
 
 There are many reasons to create a Contributor Insight Page, but here's some inspiration to get you started:
 
 - **Interacting contributors**: Keep tabs on contributors actively interacting with your repository.
 - **Organization's developers**: Monitor the contributions and performance of your internal team.
-- **VIP contributors**: Highlight and track contributors who you've identifies as key maintainers or significant contributors.
+- **VIP contributors**: Highlight and track contributors you've identified as key maintainers or significant contributors.
 - **Sales leads**: Identify contributors whose engagement could potentially be turned into sales opportunities.
 - **Potential hires**: Observe contributors who exhibit skill and commitment, making them prime candidates for hiring.
 - **Alumni contributors**: Keep an eye on contributors who have decreased their engagement or left the project.
 
 ### Creating a New Contributor Insight Page
 
-1. Click the + next to Insights in the sidebar to create a new insight page. You will be redirected to a page where you can create your new insight page.
+1. Click the "+" next to "Insights" in the sidebar.
+2. Click "New Contributor Insight." You will be redirected to a page where you can create your new Contributor Insight Page.
 
-![New Contributor Insight Page](../../static/img/contrib-insight-new.png)
+   ![New Contributor Insight Page](../../static/img/contrib-insight-new.png)
 
-2. Click on the "New Contributor Insight" to start a new insight page.
 3. Give your page a name.
 4. Choose your page visibility. You can choose to make your Contributor Insight Page public or private. By default, it is set to public.
 5. Add contributors to your page by searching for their GitHub username, syncing your GitHub Team, or importing your GitHub Following.
 
-![team sync gif](../../static/gif/team-sync.gif)
+   ![team sync gif](../../static/gif/team-sync.gif)
 
 ## Using Your Contributor Insight Page
```

Comment:
In this section, there are three tabs explained: Overview, Activity, and Contributor.
With the new workspace, we now have: Overview, Activity, and Highlights (see screenshot below).

We might want to update this as well as add a new image to replace the one in "Activity" subsection. We can add it here or create a new issue for that.

![Screenshot 2024-03-03 234339](https://github.com/open-sauced/docs/assets/45172775/c45c665f-0f3e-4cfd-b8ea-9ac042ffa774)

Reply:
> In this section, there are three tabs explained: Overview, Activity, and Contributor.
> 
> With the new workspace, we now have: Overview, Activity, and Highlights (see screenshot below).
> 
> 
> 
> We might want to update this as well as add a new image to replace the one in "Activity" subsection. We can add it here or create a new issue for that.
> 
> 
> 
> ![Screenshot 2024-03-03 234339](https://github.com/open-sauced/docs/assets/45172775/c45c665f-0f3e-4cfd-b8ea-9ac042ffa774)
> 
> 
> 
> 
> 
> 

I think creating a separate issue for this would be good @adiati98. If you & @BekahHW go with this idea, I'll create the issue for it.

Reply:
I would rather have a new issue for this, just so we can get this merged in.

Reply:
Ok I'll create the issue

Reply:
@CBID2 I've changed this to "Highlights".
I just noticed we have it in the contributor section. So, I use that one.

Reply:
Hey @adiati98. I just saw this message. I made [the issue](https://github.com/open-sauced/docs/issues/270). Since we don't need it yet, I kept it in triage mode. Is that ok?